src/scripts/main.js

```diff
@@ -1,3 +1,247 @@
 'use strict';
 
-// write your code here
+const cells = document.querySelectorAll('.field-row');
+const startButton = document.querySelector('button');
+const gameScore = document.querySelector('.game-score');
+const message = document.querySelector('.message-container');
+
+const gameField = [];
+const randomNumbers = [2, 2, 2, 2, 2, 2, 2, 2, 2, 4];
+const doublingNumbers = [];
+
+let startOrRestart = 'start';
+let sum = 0;
+let winGame = false;
+
+for (const cell of cells) {
+  for (let i = 0; i < 4; i++) {
+    gameField.push(cell.children[i]);
+  }
+}
+
+let cleanField = [...gameField];
+
+const startGame = () => {
+  if (startOrRestart === 'start') {
+    startButton.textContent = 'Restart';
+    startButton.classList = 'button restart';
+    startOrRestart = 'restart';
+    message.children[2].classList = 'message message-start hidden';
+  }
+
+  if (startOrRestart === 'restart') {
```

Comment:
It's better to use constants for all text values. It's easy to make mistake somewhere in the string